react/OrderShipping.tsx

```diff
@@ -1,31 +1,206 @@
-import React, { createContext, ReactNode, useContext } from 'react'
-
+import React, {
+  createContext,
+  ReactNode,
+  useContext,
+  useCallback,
+  useEffect,
+  useRef,
+} from 'react'
+import { compose, graphql } from 'react-apollo'
+import { useOrderQueue } from 'vtex.order-manager/OrderQueue'
 import { useOrderForm } from 'vtex.order-manager/OrderForm'
 
+import {
+  estimateShipping as EstimateShipping,
+  selectDeliveryOption as SelectDeliveryOption,
+} from 'vtex.checkout-resources/Mutations'
+
 interface Context {
   countries: string[]
+  selectedAddress: CheckoutAddress
+  insertAddress: any
+  deliveryOptions: DeliveryOption[]
+  selectDeliveryOption: any
 }
 
 interface OrderShippingProps {
   children: ReactNode
+  EstimateShipping: any
+  SelectDeliveryOption: any
 }
 
 const OrderShippingContext = createContext<Context | undefined>(undefined)
 
-export const OrderShippingProvider = ({ children }: OrderShippingProps) => {
-  const {
-    orderForm: {
-      shipping: { countries },
-    },
-  } = useOrderForm()
-
-  return (
-    <OrderShippingContext.Provider value={{ countries }}>
-      {children}
-    </OrderShippingContext.Provider>
-  )
+const shippingId = 'Shipping'
+
+const PENDING = 'Pending'
```

Comment:
Wouldn't this look better if: 
```suggestion
const STATUS = {
  PENDING:  'Pending'
  FULFILLED: 'Fulfilled'
} 
```